contracts/library/UniswapRouter.sol

```diff
@@ -20,7 +20,7 @@ library UniswapRouter {
         IERC20 startToken,
         uint256 minAmountOut,
         uint256 deadline
-    ) public returns (uint256) {
+    ) internal returns (uint256) {
```

Comment:
Setting these functions as internal allows the compiler to inline them within the deployed contract, rather than having to link them whenever we deploy a contract that requires them. There are pros and cons for both approaches, but setting them as internal makes deployment a bit easier. This is the only implementation change included in this PR. (Also happy to include this change in a separate PR instead.)